shell/platform/windows/window.cc

```diff
@@ -200,10 +201,36 @@ LRESULT Window::OnGetObject(UINT const message,
   }
 
   gfx::NativeViewAccessible root_view = GetNativeViewAccessible();
+  // TODO(schectman): UIA is currently disabled by default.
+  // https://github.com/flutter/flutter/issues/114547
+#ifdef FLUTTER_ENGINE_USE_UIA
```

Comment:
Could we move this into the `if (is_uia_request && root_view) {` and then keep the `else if (is_msaa_request ... )`?